ack/readers/awin_advertiser/config.py

```diff
@@ -0,0 +1,52 @@
+# GNU Lesser General Public License v3.0 only
+# Copyright (C) 2020 Artefact
+# licence-information@artefact.com
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 3 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public License
+# along with this program; if not, write to the Free Software Foundation,
+# Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+from datetime import datetime
+
+from pydantic import BaseModel, validator
+
+DATEFORMAT = "%Y-%m-%d"
+
+AWIN_API_ENDPOINT = "https://api.awin.com/advertisers/"
+LIMIT_NVIEWS_PER_REQ = 5
+MAX_WAIT_REPORT_DELAY = 4096
+
+# https://wiki.awin.com/index.php/Advertiser_Service_API
+REPORT_TYPES = [
+    "publisher",
+    "creative",
+    "campaign"
+]
+
+
+class AwinAdvertiserReaderConfig(BaseModel):
+    auth_token: str
+    advertiser_id: str
+    report_type: str
```

Comment:
For field like this, you can enforce the value to be one of the REPORT_TYPES value. You just have to write `report_type: Literal[tuple(REPORT_TYPES)]`.
I also saw that in the cli file you put a default value for report_type so you can add it here also (if it's logical to you)